### PR TITLE
123 Gridster inside gridster

### LINKED ISSUE #123: Gridster inside gridster
It's possible to insert a `<gridster>` inside a `<gridster-item>` element?

Something like:

`<gridster> <!-- outter -->`
  `<gridster-item> <!-- item from outter -->`
    `<gridster> <!-- inner -->`
      `<gridster-item> <!-- item from inner gridster -->`
     `</gridster-item>`
   `</gridster>`
  `</gridster-item>`
`</gridster>`

sry for the confusion :x

### PR DESCRIPTION
**Pull request by swiety85**
Resolves #123

### CODE REVIEW

**Comment by richardlnnr** [comment]
In my tests, it's all working well :-)

File: src/gridster-prototype/gridster-prototype.service.ts [hunk 1]
```diff
@@ -183,25 +208,23 @@ export class GridsterPrototypeService {
...
-     * @param containerEl Element above which "element" is dragged
-     * @param event DraggableEvent
-     * @returns {boolean}
      */
-    private isOverGridster(item: GridsterItemPrototypeDirective, gridster: GridsterService, event): boolean {
+    private isOverGridster(item: GridsterItemPrototypeDirective, gridsterEl: HTMLElement, event, options): boolean {
         const el = item.$element;
-        const elContainer = gridster.gridsterComponent.$element;
-        const tolerance = gridster.options.tolerance;
+        const parentItem = <HTMLElement>gridsterEl.parentElement.closest('gridster-item');
```
Comment (richardlnnr) [inline src/gridster-prototype/gridster-prototype.service.ts]: Sometimes, the parentElement on gridsterEl are null. I reported that on the issue #226

Comment (richardlnnr) [inline src/gridster-prototype/gridster-prototype.service.ts]: Sometimes, the parentElement on gridsterEl are null. I reported that on the issue #226

**Comment by swiety85** [comment]
I will do it tomorrow. Thx for patience. 😊

**Pull request merged**